packages/origin-ui-core/src/features/users/sagas.ts

```diff
@@ -103,6 +104,7 @@ function* fetchOffchainUserDetails(): SagaIterator {
                     organization
                 })
             );
+            yield put(reloadCertificates());
```

Comment:
Is that the correct approach? What is the root cause of having wrong data after logout/login sequence?

Reply:
The reason is that bundles are not refetched after user login. `reloadCertificates` action triggers this reloading along with reloading of the certificates and orders as it would be better to synchronize their state